python/python/lancedb/query.py

```diff
@@ -136,7 +137,9 @@ def create(
 
         if isinstance(query, str):
             # fts
-            return LanceFtsQueryBuilder(table, query)
+            return LanceFtsQueryBuilder(
```

Comment:
this is only place we use it, we could assert if we hit another path and the ordering_field_name is not defined or log a warning. Or just ignore as we do here (kinda like the vector column name field). Not sure what you all prefer as a standard, so left like this.